### ISSUE TITLE
deal with flat channels

### ISSUE THREAD

**Issue by jasmainak** [autoreject/autoreject#55]
@jbschiratti reported to me that he has a use case for running autoreject on his data with flat channels. He has locally bad channels and MNE gets overzealous in rejecting trials. Perhaps we should support this as well.

**Comment by agramfort** [comment]
it's unclear how. But as we said in the paper flat channels are trivial to
detect. Altough we don't have a function for that...

**Comment by jasmainak** [comment]
yes detection is easier for flat channels. But you still can't repair 
locally bad sensors using epoch-wise interpolation in MNE.

**Comment by agramfort** [comment]
true :(

there is a PR though to have epoch wise interpolation. Maybe this could
become a priority to fix...
